docs/pages/versions/unversioned/sdk/device.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Device
+---
+
+Provide information of devices on the application.
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. It is not yet available for [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native apps.
+
+## API
+
+```js
+import { Device } from 'expo';
```

Comment:
Should we advertise this way of importing in new documentation?

Reply:
Would 'import * as Device from 'expo-device';' better than this?

Reply:
what alternative would you suggest?

Reply:
indeed, `import * as Device from 'expo-device'` is preferred. in sdk34 we will remove imports of this style from the expo package

Reply:
What would the alternative be?